### ISSUE TITLE
Setup benchmark CI for KEVM

### ISSUE THREAD

**Issue by palinatolmach** [runtimeverification/kontrol#10]
We should set up the CI for running `kevm foundry-prove` in the tooling benchmark repository and add out tests there: https://github.com/eth-sc-comp/benchmarks.

- [ ] Integrate conformance tests used in KEVM (https://github.com/runtimeverification/evm-semantics/tree/master/tests/foundry)
- [ ] Prepare scripts for `foundry-prove`

**Comment by palinatolmach** [comment]
@JuanCoRo do you think setting up the repo for Solady falls into the scope of this issue?

**Comment by JuanCoRo** [comment]
Absolutely! Using that repo to benchmark speed improvements with Kontrol sounds great.

**Comment by palinatolmach** [comment]
Oh wait, @JuanCoRo, sorry, I thought I commented on a different issue related to downstream testing (this one: https://github.com/runtimeverification/kontrol/issues/17, and I'll assign it to you for now, if you don't mind). I agree it makes sense too though!
